app/models/ConferenceDescriptor.scala

```diff
@@ -257,6 +257,11 @@ object ConferenceDescriptor {
     val NEUILLY_231_232 = Room("neu_232_232", "Neuilly 231M-232M", 32, "classroom", "rien", "231+232M", Some("Mezzanine South Neuilly"))
     val NEUILLY_234_235 = Room("neu_234_235", "Neuilly 234M-235M", 24, "classroom", "rien", "234+235M", Some("Mezzanine South Neuilly"))
 
+    val NEUILLY_LAB1 = Room("lab1", "Neuilly LAB 1", 30, "classroom", "rien", "LAB 1", Some("Neuilly"))
+    val NEUILLY_LAB2 = Room("lab2", "Neuilly LAB 2", 30, "classroom", "rien", "LAB 2", Some("Neuilly"))
+    val PARIS_LAB3 = Room("lab3", "Paris LAB 3", 30, "classroom", "rien", "LAB 3", Some("Paris"))
+    val PARIS_LAB4 = Room("lab4", "Paris LAB 4", 30, "classroom", "rien", "LAB 4", Some("Paris"))
```

Comment:
```suggestion
    val PARIS_LAB4 = Room("lab4", "Paris LAB 4", 20, "classroom", "rien", "LAB 4", Some("Paris"))
```

@nicmarti maybe 20 as shared by @agoncal

Reply:
ok